deepchem/feat/graph_features.py

```diff
@@ -668,6 +687,11 @@ def __init__(self, master_atom=False, use_chirality=False,
       provided in atom_properties.  So "atom 00000000 sasa" would be the
       name of the molecule level property in mol where the solvent
       accessible surface area of atom 0 would be stored.
+    per_atom_fragmentation: Boolean
+      If True, then multiple "atom-depleted" featurizations will be possible to do for each molecule. It will be
+      possible to remove atoms  one by one, and then, featurize each atom-depleted molecule.
+      Thus, applying featurize method  will produce a set of ConvMol objects for each molecule. This is useful for
+      subsequent model interpretation: finding atoms favorable/unfavorable for (modelled) activity.
```

Comment:
Can we reword this slightly?  Saying that it "will be possible" isn't quite what you mean.  How about this:

> If True, then multiple "atom-depleted" versions of each molecule will be created.  For each molecule, atoms are removed one at a time and the resulting molecule is featurized.  The result is a list of ConvMol objects, one with each heavy atom removed.  This is useful for subsequent model interpretation: finding atoms favorable/unfavorable for (modelled) activity.  This option is typically used in combination with a FlatteningTransformer to split the lists into separate samples.

Reply:
ok